dvc/remote/__init__.py

```diff
@@ -26,21 +35,30 @@
 ]
 
 
-def _get(remote_conf):
-    for remote in REMOTES:
+def _get(remote_conf, remotes, default):
```

Comment:
@efiop I think cleaning this up to use `tree.supported(...)` will also require moving scheme and the remaining constants into the tree classes like we discussed. And then once we have single `Remote` and `Cache` classes that take a tree as a parameter, we'll be able to get rid of these helper methods.

I think it would be best to leave this as-is for now, and then handle it in the next PR